corehq/apps/ota/tests/test_search_claim_endpoints.py

```diff
@@ -42,7 +42,10 @@ def setUp(self):
             case_id=self.case_id,
             case_type=CASE_TYPE,
             case_name=CASE_NAME,
+            external_id=CASE_NAME,
+            user_id=OWNER_ID,
             owner_id=OWNER_ID,
+            update={'opened_by': OWNER_ID},
```

Comment:
i'm surprised this was necessary. doesn't the case processing framework set this automatically based on the id of the case creator?